### DISCUSSION TITLE
Stuck/error on first deployment

### DISCUSSION THREAD

**Discussion by StefanMPopp** [alshedivat/al-folio#2227] [Q&A] [answered]
I just followed the instructions with the docker image and changed a few things in the `_config.yml`
When trying to deploy with `docker-compose up` (Source: `deploy from a branch`, `master` are set), it is stuck at:
```
...
from /usr/local/bin/jekyll:25:in `load'
from /usr/local/bin/jekyll:25:in `<main>'
```

When I try to deploy it from GitHub actions, it throws me the following error:
```
...
from /opt/hostedtoolcache/Ruby/3.0.2/x64/bin/bundle:23:in `load'
from /opt/hostedtoolcache/Ruby/3.0.2/x64/bin/bundle:23:in `<main>'
Error: Process completed with exit code 1.
```

Any help is highly appreciated!

**Comment by george-gca** [comment, marked as answer]
We first recommend you change the bare minimum to make it work:
- `url` and `baseurl` for deploying to GitHub pages
- nothing at all when testing it locally

From that point on add as little changes as possible and test between them to check if everything is working. This is the easiest way to ensure your code works and also which introduced change breaks it, if it happens.
